Sources/PenguinParallel/NonblockingThreadPool/NonBlockingThreadPool.swift

```diff
@@ -289,7 +289,49 @@ public class NonBlockingThreadPool<Environment: ConcurrencyPlatform>: ComputeThr
     if let e = err { throw e }
   }
 
-  /// Shuts down the thread pool.
+  /// Executes `fn`, optionally in parallel, spanning the range `0..<n`.
+  public func parallelFor(n: Int, _ fn: VectorizedParallelForBody) {
```

Comment:
Not a fan of this doc.  When you say “optionally” it implies the caller gets to pass an option, but that's not the case.  Also, I have no idea what it means to “execute a function spanning a range.”  Was going to make some suggestions but you've already merged.